packages/markdown/parseMarkdown.js

```diff
@@ -447,6 +447,12 @@ function resolveComponentApiUrl(productId, componentPkg, component) {
   if (productId === 'x-date-pickers') {
     return `/x/api/date-pickers/${kebabCase(component)}/`;
   }
+  if (productId === 'x-charts') {
+    return `/x/api/charts/${kebabCase(component)}`;
```

Comment:
Shouldn't it also include the ending slash, the same as for the pickers and general API case? 🤔 
```suggestion
    return `/x/api/charts/${kebabCase(component)}/`;
```

Reply:
Nice catch, thanks!